### DISCUSSION TITLE
zowe tso issue accounNumber type

### DISCUSSION THREAD

**Discussion by kraabrsg** [zowe/zowe-cli#1342] [Q&A] [answered]
Hi folks,

in the zowe tsoissuecommand example there is the accountNumber defined as IProfile, but in the class "IssueTso" the accountNumber is declared as string. 
When doing
`accountNumber = defaultTsoProfile.account ;`
account is of type any, so i choose string, is this correct ? (We are on Zowe 6.37)

Thanks!

**Comment by t1m0thyj** [comment, marked as answer]
You are correct, `account` is defined as a string in the TSO profile schema:
https://github.com/zowe/zowe-cli/blob/aa550ddaa14dfbce76935cf5e8352e9ba1046f3d/packages/cli/src/imperative.ts#L287-L290
